extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/config/JsonConfig.java

```diff
@@ -0,0 +1,52 @@
+package io.quarkus.micrometer.runtime.config;
+
+import java.time.Duration;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class JsonConfig implements MicrometerConfig.CapabilityEnabled {
+    /**
+     * Support for export to JSON format. Off by default.
+     */
+    @ConfigItem(defaultValue = "false")
+    public Optional<Boolean> enabled;
+
+    /**
+     * The path for the JSON metrics endpoint.
+     * The default value is {@code /metrics-json}.
+     */
+    @ConfigItem(defaultValue = "/metrics")
```

Comment:
Javadoc doesn't match the actual config default. Which should it be?

Reply:
Good catch, changed the javadoc to `/metric`

Reply:
Thanks!